### PR TITLE
Doc: Readme updates to align with new Red Hat template

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

README at the root of the collection has been updated to conform with the new [Red Hat collection template](https://access.redhat.com/articles/7068606). Due to not being able to link to files outside of the Mkdocs `docs_dir` location, the old README at the root of the collection has been renamed to `index.md`. This is used as the homepage on the documentation site. The root README for the repository is only used for GitHub.

## Proposed changes

Please see the change summary.

## How to test

No updates to main site docs.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by JulioPDX** [comment]
pre-commit.ci run

File: ansible_collections/arista/cvp/README.md [hunk 1]
```diff
@@ -4,112 +4,66 @@
...
-### Python
-
-- Python `>=3.9`
-
-Please check the minimum version supported by your ansible installation on the [ansible website](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#control-node-requirements).
+The CVP collection has the following requirements:
 
-### Additional Python Libraries required
+- Python 3.9+
+- Ansible Core 2.14.0 to 2.16.x
```
Comment (carlbuchmann) [inline ansible_collections/arista/cvp/README.md]:
```suggestion
- Ansible Core 2.15.0 to 2.17.x
```

File: ansible_collections/arista/cvp/README.md [hunk 2]
```diff
@@ -4,112 +4,66 @@
...
-Installing 'arista.cvp:1.1.0' to '~/.ansible/collections/ansible_collections/arista/cvp'
+ansible-galaxy collection install arista.cvp --upgrade
 ```
 
-Complete installation process is available on [repository website](docs/installation/requirements/)
-
-## Collection overview
-
-This repository provides content for Ansible's collection **arista.cvp** with following content:
+You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 4.7.1:
```
Comment (carlbuchmann) [inline ansible_collections/arista/cvp/README.md]:
```suggestion
You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 3.10.1:
```

File: ansible_collections/arista/cvp/README.md [hunk 3]
```diff
@@ -187,18 +141,22 @@ ansible_httpapi_port=443
...
 Contributing pull requests are gladly welcomed for this repository. If you are planning a significant change, please start a discussion first to ensure we can merge it.
 
 You can also open an [issue](https://github.com/aristanetworks/ansible-cvp/issues) to report any problem or to submit enhancement.
 
-A more complete [guide for contribution](https://avd.sh/en/latest/docs/contribution/overview.html) is available in the repository
+A more complete [guide for contribution](hhttps://avd.arista.com/stable/docs/contribution/overview.html) is available.
+
+## Support
+
+Support for this `arista.cvp` collection is provided by the community directly in this repository. If you have any questions, please open an [issue](https://github.com/aristanetworks/ansible-cvp/issues).
```
Comment (carlbuchmann) [inline ansible_collections/arista/cvp/README.md]:
```suggestion
- The `arista.avd` Ansible collection version 4.x releases and higher offer full support from Arista TAC, which includes the `arista.cvp` Ansible collection. If your organization has the [A-Care subscription](https://www.arista.com/assets/data/pdf/AVD-A-Care-TAC-Support-Overview.pdf) please don't hesitate to contact TAC with any questions or issues.
- Community support is provided via [Github discussions board](https://github.com/aristanetworks/ansible-cvp/discussions).
```

File: ansible_collections/arista/cvp/README.md [hunk 4]
```diff
@@ -187,18 +141,23 @@ ansible_httpapi_port=443
...
+Please see our [pipeline workflows](https://github.com/aristanetworks/ansible-cvp/actions) to view the in-depth testing performed against the collection.
 
 ## Contributing
 
 Contributing pull requests are gladly welcomed for this repository. If you are planning a significant change, please start a discussion first to ensure we can merge it.
 
 You can also open an [issue](https://github.com/aristanetworks/ansible-cvp/issues) to report any problem or to submit enhancement.
 
-A more complete [guide for contribution](https://avd.sh/en/latest/docs/contribution/overview.html) is available in the repository
+A more complete [guide for contribution](hhttps://avd.arista.com/stable/docs/contribution/overview.html) is available.
```
Comment (carlbuchmann) [inline ansible_collections/arista/cvp/README.md]:
```suggestion
```

File: ansible_collections/arista/cvp/README.md [hunk 5]
```diff
@@ -187,18 +141,22 @@ ansible_httpapi_port=443
...
+Please see our [pipeline workflows](https://github.com/aristanetworks/ansible-cvp/actions) to view the in-depth testing performed against the collection.
 
 ## Contributing
 
 Contributing pull requests are gladly welcomed for this repository. If you are planning a significant change, please start a discussion first to ensure we can merge it.
 
 You can also open an [issue](https://github.com/aristanetworks/ansible-cvp/issues) to report any problem or to submit enhancement.
 
-A more complete [guide for contribution](https://avd.sh/en/latest/docs/contribution/overview.html) is available in the repository
+
```
Comment (carlbuchmann) [inline ansible_collections/arista/cvp/README.md]:
```suggestion
```

**Review by sugetha24 — APPROVED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**